packages/core/test/execute.spec.ts

```diff
@@ -31,6 +31,7 @@ describe('execute', () => {
     expect(spiedPlugin.spies.afterResolver).toHaveBeenCalledTimes(3);
     expect(spiedPlugin.spies.afterExecute).toHaveBeenCalledTimes(1);
     expect(spiedPlugin.spies.afterExecute).toHaveBeenCalledWith({
+      args: expect.any(Object),
```

Comment:
I think in long term it would be useful to create custom matchers for the main types, like execution arguments, then we could just do `args: expect.toBeExecutionArguments` or something like that

Reply:
Agree!